packages/libs/react-ui/src/components/Table/Table.tsx

```diff
@@ -3,15 +3,19 @@ import { TBody } from './TBody';
 import { THead } from './THead';
 import { CompoundType } from './types';
 
+import classNames from 'classnames';
 import React, { FC } from 'react';
 
 export interface ITableProps {
   children?: CompoundType<typeof TBody> | CompoundType<typeof THead>;
+  striped?: boolean;
 }
 
-export const Table: FC<ITableProps> = ({ children }) => {
+export const Table: FC<ITableProps> = ({ children, striped }) => {
   return (
-    <table className={tableClass}>
+    <table
+      className={striped ? classNames(tableClass, 'stripedClass') : tableClass}
```

Comment:
nit: just a small improvement
```suggestion
      className={classNames(tableClass, { stripedClass: striped })}
```

Reply:
Good suggestion, didnt know that was posible, applied 👍